src/app/Models/MenuItem.php

```diff
@@ -27,14 +28,24 @@ public function page()
         return $this->belongsTo('Backpack\PageManager\app\Models\Page', 'page_id');
     }
 
+    public function menu()
+    {
+        return $this->belongsTo('Backpack\MenuCRUD\app\Models\Menu', 'menu_id', 'id');
+    }
+
     /**
      * Get all menu items, in a hierarchical collection.
      * Only supports 2 levels of indentation.
      */
-    public static function getTree()
+    public static function getTree($menuName = null)
     {
         $menu = self::orderBy('lft')->get();
 
+        if(!is_null($menuName)) {
+            $menuId = Menu::where('name', $menuName)->first()->id;
```

Comment:
Will this throw some sort of error if `Menu::where('name', $menuName)->first()` happens to return `null` (i.e. not found)?